corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -870,6 +870,21 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         print: function () {
             window.print();
         },
+        onRender: function() {
+            const self = this;
+            $.getJSON('/langcodes/langs.json', function (mapping) {
```

Comment:
isn't this already or if could be made available on HQ page context itself? So don't we don't need to send a request for this separately?

Reply:
https://github.com/dimagi/commcare-hq/pull/33158/commits/fd47c981452221abe96b2a63591698611a2cfaed